grid_map_core/include/grid_map_core/GridMap.hpp

```diff
@@ -381,6 +381,12 @@ class GridMap
    */
   void clearAll();
 
+  /*!
+  * Clears cells that are set to 1.0 in the layer "clear".
```

Comment:
According to the code below, the cells are not only cleared but overwritten with the values from the new submap. Can you describe this?